docs/examples/physics_using_pybullet/viz_domino.py

```diff
@@ -0,0 +1,238 @@
+
+brick_colors = np.random.rand(nb_bricks, 3)
+
+brick_coll = p.createCollisionShape(p.GEOM_BOX,
+                                    halfExtents=brick_size / 2)
+
+# We use this array to store the reference of brick objects in pybullet world.
+bricks = np.zeros(nb_bricks, dtype=np.int8)
+
+
+n_dominoes = 10
+centers_list = np.zeros((n_dominoes, 3))
+
+# Adding the dominoes
+for i in range(n_dominoes):
+    center_pos = np.array([(i*0.99)-2.8,1,0])
+    brick_centers[i] = center_pos
+    brick_orns[i] = np.array([0, 0, 0, 1])
+    bricks[i] = p.createMultiBody(baseMass=brick_mass,
+                                      baseCollisionShapeIndex=brick_coll,
+                                      basePosition=center_pos,
+                                      baseOrientation=brick_orns[i])
+    p.changeDynamics(bricks[i], -1, lateralFriction=0.1, restitution=0.1)
+
+
+brick_actor = actor.box(centers=brick_centers,
+                        directions=brick_directions,
+                        scales=brick_sizes,
+                        colors=brick_colors)
+
+
+# Now, we define a scene and add actors to it.
+scene = window.Scene()
+scene.add(actor.axes())
+scene.add(base_actor)
+scene.add(brick_actor)
+
+# Create show manager.
+showm = window.ShowManager(scene,
+                           size=(900, 768), reset_camera=False,
+                           order_transparent=True)
+
+showm.initialize()
+
+# Counter iterator for tracking simulation steps.
+counter = itertools.count()
+
+# Variable for tracking applied force.
+apply_force = True
+
+
+# Now, we define methods to sync objects between fury and Pybullet.
+
+# Get the position of base and set it.
+base_pos, _ = p.getBasePositionAndOrientation(base)
+base_actor.SetPosition(*base_pos)
+
+
+# Calculate the vertices of the bricks.
+vertices = utils.vertices_from_actor(brick_actor)
+num_vertices = vertices.shape[0]
+num_objects = brick_centers.shape[0]
+sec = np.int(num_vertices / num_objects)
+
+
+"""
+==============
+Syncing Bricks
+==============
+
+Here, we perform three major steps to sync bricks accurately.
+* Get the position and orientation of the bricks from pybullet.
+* Calculate the Rotation Matrix.
+  * Get the difference in orientations (Quaternion).
+  * Generate the corresponding rotation matrix according to that difference.
+  * Reshape it in a 3x3 matrix.
+* Perform calculations to get the required position and orientation.
+* Update the position and orientation.
+"""
+
+def sync_brick(object_index, multibody):
+    pos, orn = p.getBasePositionAndOrientation(multibody)
+
+    rot_mat = np.reshape(
+        p.getMatrixFromQuaternion(
+            p.getDifferenceQuaternion(orn, brick_orns[object_index])),
+        (3, 3))
+
+    vertices[object_index * sec: object_index * sec + sec] = \
+        (vertices[object_index * sec: object_index * sec + sec] -
+         brick_centers[object_index]) @ rot_mat + pos
+
+    brick_centers[object_index] = pos
+    brick_orns[object_index] = orn
```

Comment:
This method is responsible to update the position and orientation of the objects. So if according to physics the domino is supposed to be stationary then it will be stationary. So you can safely pass the first domino here.